lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = @kernelSpec.language
```

Comment:
We need to remove [this line](https://github.com/nteract/hydrogen/blob/e3927895dc903d408e5fe1624ff3e81cdaed3b59/lib/signal-list-view.coffee#L63) as well.